packages/server/tests/acceptance.spec.ts

```diff
@@ -374,13 +383,23 @@ describe('RPC Server Acceptance Tests', async function () {
     });
 
     it('should execute "eth_getBalance" for newly created account with 5000 HBAR', async function () {
-        const { accountInfo } = await utils.createEthCompatibleAccount(client, null, 5000);
+        const accountResponse = await utils.createEthCompatibleAccount(client, null, 5000);
+        const accountInfo = accountResponse.accountInfo;
         const mirrorResponse = await utils.callMirrorNode(`accounts/${accountInfo.accountId}`);
         const mirrorAccount = mirrorResponse.data;
         const res = await utils.callSupportedRelayMethod('eth_getBalance', [mirrorAccount.evm_address, 'latest']);
         expect(res.data.result).to.eq('0x10f0777f464e77a2400');
     });
 
+    it('should execute "eth_getBalance" with account alias', async function () {
+        const accountResponse = await utils.createEthCompatibleAccount(client, null, 5000);
```

Comment:
Q: could you use mirrorAccount3 here or what's the reason to create another account?

Reply:
Okay. Just want to reduce cost run but this is fine for now